### DISCUSSION TITLE
Pressing Ctrl+Shift+F on a repository page tries to open Code Search, but is blocked on Firefox

### DISCUSSION THREAD

**Discussion by penguinencounter** [community/community#39024] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

On Chrome, the functionality works as expected, but on Firefox, the window is blocked.
![Firefox prevented this site from opening a popup window.](https://user-images.githubusercontent.com/49845522/201549002-f561833d-dcd1-4ee6-b9f6-2b523144935f.png)

**Comment by look** [comment, marked as answer]
Sorry about that! We [recently integrated the functionality of cs.github.com into GitHub itself](https://github.blog/changelog/2022-11-09-introducing-an-all-new-code-search-and-code-browsing-experience/), which means you no longer need to go to the other site to search. You should have access since you were in the Technology Preview for cs.github.com, so check your `Feature previews` under your settings and turn it on. If you _don't_ have access, you can [sign up for the wait list](https://github.com/features/code-search) and I will make sure you are added.
